File: AULA_02/Haskell/Triangulo.hs
triangulo x y z
         | not ((x+y)>z && (x+z)>y && (y+z)>x) = "Não é triângulo"
         | x==y && y==z = "Equilatero"
         | x==y || y==z = "Isóceles"
         | otherwise = "Escaleno"

main = do
      putStrLn "Digite as dimensões: "
      line <- getLine
      let x = read line
      line <- getLine
      let y = read line
      line <- getLine
      let z = read line

      putStrLn $ triangulo x y z

File: AULA_02/C/Triangulo.c
#include <stdio.h>

int main() {
  float x,y,z;

  printf("Entre com as 3 dimensões: ");
  scanf("%f %f %f",&x,&y,&z);

  if( x+y>z && x+z>y && z+y>x ){
    if( x==y && y==z ){
      printf("Equilatero\n");
    }else if( x==y || y==z || x==z ){
      printf("Isóceles\n");
    }else{
      printf("Escaleno\n");
    }
  }else{
    printf("Não é triangulo\n");
  }

  return 0;
}
